packages/main/src/DatePicker.ts

```diff
@@ -59,6 +59,11 @@ type DatePickerChangeEventDetail = {
 	valid: boolean,
 }
 
+type ValueStateChangeEventDetail = {
```

Comment:
As with all event details, we need to export them, so that he TS type is available for consumers that would like to make use of typescript checking. The rest of the code seems fine to me.